models/service.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const ServiceSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  streetAddress: {
+    type: String,
+    required: true
+  },
+  city: {
+    type: String,
+    required: true
+  },
+  state: {
+    type: String,
+    required: true
+  },
+  zipCode: {
+    type: String,
+    required: true
+  },
+  phone: {
+    type: String,
+    required: true
+  },
+  type: {
+    type: String,
+    required: true
+  }
+});
+
+ServiceSchema.index({ 'name': 'text' });
+
+const Service = mongoose.model('service', ServiceSchema);
+
+module.exports = Service;
```

Comment:
I also can't remember if our resources page was going to be a link to one that SJI already provides? Is that a thing that exists?